infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -101,7 +101,7 @@
      * per streams map to last issued global-log position. used for
      * backpointers.
      */
-    private final ConcurrentHashMap<UUID, Long> streamTailToGlobalTailMap = new
+    private final ConcurrentHashMap<UUID, Long> streamTails = new
```

Comment:
just curious why the name change. Also please update comment to reflect name change

Reply:
anny pointed out that it wasn't intuitive in the last PR.

Reply:
Maybe we can leave the name change to a different PR?